beetsplug/info.py

```diff
@@ -52,7 +53,11 @@ def emitter():
             tags[field] = getattr(mf, field)
         tags['art'] = mf.art is not None
         tags['path'] = displayable_path(path)
+        # create a temporary Item to take advantage of __format__
+        tags['item'] = Item(db=None, **tags)
```

Comment:
This might be a bit fragile and up for debate. The rationale behind it is to have an `Item` be available for making use of `Item.__format__` during `print_data()`.

It might be cleaner to modify the emitters so they return an `(Item, data)` tuple instead of packing it inside `data['item']` as well.